.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Proposed changes
+
+Describe your changes here. Why are they necessary? If it fixes a bug or
+resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
+- [ ] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
+- [ ] Linting checks pass with my changes.
+- [ ] Any existing unit tests pass with my changes.
+- [ ] Any existing end-to-end tests pass with my changes.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
+- [ ] I have added JSDoc comments as appropriate.
+- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
+
```

Comment:
- [ ] I have shared this PR on channel the roar-pr-reviews
 - [ ] I have linked relevant issues(if any)